src/main/flight/gps_rescue.c

```diff
@@ -140,7 +140,9 @@ PG_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig,
     .minSats = 8,
     .minRescueDth = 100,
     .allowArmingWithoutFix = false,
-    .useMag = true
+    .useMag = true,
+    .targetLandingAltitudeM = 7,
```

Comment:
Why the odd default value of 7 meters?

Reply:
It is only a mid value(3-10) for example to avoid some trees on the way back.